packages/playground-examples/copy/fa/JavaScript/README.md

```diff
@@ -0,0 +1,3 @@
+## مثال های جاوااسکریپت
+
+این مثال ها به ما چگونگی اجرای جاوااسکریپت توسط تایپ اسکریپت را نشان می دهند. با اینکه در بعضی از مثال ها از تایپ قابلیت های ساده تایپ اسکریپت استفاده می کنیم اما هدف اصلی این دسته از مثال ها نشان دادن سادگی استفاده مجدد از دانسته های کاربر در جاوااسکریپت در تایپ اسکریپت است.
```

Comment:
Use ZWNJ for TypeScript. According to Persian standards, you should not use space for those cases.